### DISCUSSION TITLE
HHS页面全屏后左边导航栏消失

### DISCUSSION THREAD

**Discussion by turing-bird** [1Panel-dev/1Panel#5461] [Q&A]
### 1Panel 版本

v1.10.10-lts

### 问题描述

HHS页面点了全屏，退出后左边导航栏消失。重启服务器 菜单栏都不会出来！

### 重现步骤

![微信截图_20240614160353](https://github.com/1Panel-dev/1Panel/assets/52934646/daaf8b88-f2d7-453a-ab07-487cb8b2219e)
重启服务器 菜单栏都不会出来！

**Comment by wanghe-fit2cloud** [comment]
尝试清除浏览器缓存或者更换其他浏览器。
